### PR TITLE
Update lematerial.md

### PR DESCRIPTION
**Pull request by lritchie-entalpic**
Fixes in wording + Citations

### CODE REVIEW

File: lematerial.md [hunk 1]
```diff
@@ -214,7 +216,7 @@ LeMaterial aims to be a community-driven initiative that gathers machine learnin
               </caption>
             </table>
         
-        Its important to note here that Pymatgen's StructureMatcher failed on this example, identifying these two unit cells as different materials, when they are indeed the exact same structures. Here our hashing algorithm was able to identify them as indeed the same.
+        It is important to note here that Pymatgen's StructureMatche identified these two unit cells as different materials, when they are indeed the exact same structures. Here, our hashing algorithm was able to identify them as indeed the same.
```
Comment (inelgnu) [inline lematerial.md:219]:
```suggestion
        It is important to note here that Pymatgen's StructureMatcher identified these two unit cells as different materials, when they are indeed the exact same structures. Here, our hashing algorithm was able to identify them as indeed the same.
```

File: lematerial.md [hunk 2]
```diff
@@ -69,8 +71,8 @@ This fragmented landscape makes it challenging for researchers in AI4Science and
...
+| v.1.0 | <ul><li><strong>Data collection and merging</strong> from various sources: Materials Project, Alexandria and OQMD datasets, including multiple fields and DFT functionals (PBE, PBESol, SCAN). </li><li><strong>Data cleaning</strong>: identify and remove datapoints not conforming to the set standard (e.g. with non-compatible calculations). </li><li><strong>Standardization</strong>: uniformly format the set of fields across databases, using Optimade standard </li><li><strong>Material fingerprint</strong>: attribute a unique identifier to each material with a well benchmarked hashing function, enabling to remove duplicates</li><li><strong>Accessibility and visualization</strong>: the proposed new datasets can be visualized thanks to open source tools (phase diagrams, material explorer), thanks to Crystal Toolkit, Pymatgen, and Dash components developed by Materials Project </li></ul>  | Dec. 10, 2024 |
```
Comment (clefourrier) [inline lematerial.md:74]: Edit introduces a redundancy

File: lematerial.md [hunk 3]
```diff
@@ -214,7 +216,7 @@ LeMaterial aims to be a community-driven initiative that gathers machine learnin
               </caption>
             </table>
         
-        Its important to note here that Pymatgen's StructureMatcher failed on this example, identifying these two unit cells as different materials, when they are indeed the exact same structures. Here our hashing algorithm was able to identify them as indeed the same.
+        It is important to note here that Pymatgen's StructureMatcher identified these two unit cells as different materials, when they are indeed the exact same structures. Here, our hashing algorithm was able to identify them as indeed the same.
```
Comment (clefourrier) [inline lematerial.md:219]: Why do you want to remove that it failed?

File: lematerial.md [hunk 4]
```diff
@@ -224,10 +226,17 @@ As a community, we often place a lot of value in the **quality** of these large-
...
+Thus, if you use materials data which include (”mp-”) in the immutable_id, please cite the [Materials Project](https://pubs.aip.org/aip/apm/article/1/1/011002/119685/Commentary-The-Materials-Project-A-materials). If you use materials data which include (”agm-”) in the immutable_id, please cite [Alexandria, PBE](https://www.science.org/doi/10.1126/sciadv.abi7948) or [Alexandria PBESol, SCAN](https://www.nature.com/articles/s41597-022-01177-w). If you use materials data which include (”oqmd-”) in the immutable_id, please cite [OQMD](https://link.springer.com/article/10.1007/s11837-013-0755-4). Finally, if you make use of the Phase Diagram for visualization purposes, or the crystal viewer in the Materials Explorer, please acknowledge [Crystal Toolkit](https://github.com/materialsproject/crystaltoolkit).
+
 
 > We extend our heartfelt thanks to [Zachary Ulissi](https://zulissi.github.io/) and [Luis Barroso-Luque](https://www.linkedin.com/in/luis-barroso-luque-9742598a/) (Meta), and [Matt McDermott](https://github.com/mattmcdermott) (Newfound Materials, Inc.) for their valuable feedback regarding is initiative.
->
```
Comment (clefourrier) [inline lematerial.md:229]: Would put this section before the citation one

File: lematerial.md (hunk 2 above)
Comment (inelgnu) [reply lematerial.md:74]: @clefourrier sorry, I don't get your point, what redundant?

Comment (clefourrier) [reply lematerial.md:74]: "thanks to open source tools, thanks to ..."

File: lematerial.md [hunk 5]
```diff
@@ -224,10 +226,17 @@ As a community, we often place a lot of value in the **quality** of these large-
...
+Thus, if you use materials data which include (”mp-”) in the immutable_id, please cite the [Materials Project](https://pubs.aip.org/aip/apm/article/1/1/011002/119685/Commentary-The-Materials-Project-A-materials). If you use materials data which include (”agm-”) in the immutable_id, please cite [Alexandria, PBE](https://www.science.org/doi/10.1126/sciadv.abi7948) or [Alexandria PBESol, SCAN](https://www.nature.com/articles/s41597-022-01177-w). If you use materials data which include (”oqmd-”) in the immutable_id, please cite [OQMD](https://link.springer.com/article/10.1007/s11837-013-0755-4). Finally, if you make use of the Phase Diagram for visualization purposes, or the crystal viewer in the Materials Explorer, please acknowledge [Crystal Toolkit](https://github.com/materialsproject/crystaltoolkit).
+
 
 > We extend our heartfelt thanks to [Zachary Ulissi](https://zulissi.github.io/) and [Luis Barroso-Luque](https://www.linkedin.com/in/luis-barroso-luque-9742598a/) (Meta), and [Matt McDermott](https://github.com/mattmcdermott) (Newfound Materials, Inc.) for their valuable feedback regarding is initiative.
```
Comment (inelgnu) [inline lematerial.md:239]:
```suggestion
```

File: lematerial.md (hunk 3 above)
Comment (msiron-entalpic) [reply lematerial.md:219]: hey @clefourrier the language is too harsh, we don't want to put down other researchers in the field -- based on discussions from feedback from the community

File: lematerial.md [hunk 6]
```diff
@@ -224,10 +226,17 @@ As a community, we often place a lot of value in the **quality** of these large-
 
 **We are just getting started** — we know there are still flaws and improvements to be made — and would thus love to hear your feedback! So please reach out if you are interested to contribute to this open-source initiative. We would be excited to continue expanding LeMaterial with new datasets, tools, and applications — alongside the community! ⚛️🤗
```
Comment (inelgnu) [inline lematerial.md:228]:
```suggestion
> We extend our heartfelt thanks to [Zachary Ulissi](https://zulissi.github.io/) and [Luis Barroso-Luque](https://www.linkedin.com/in/luis-barroso-luque-9742598a/) (Meta), and [Matt McDermott](https://github.com/mattmcdermott) (Newfound Materials, Inc.) for their valuable feedback regarding is initiative.
```

File: lematerial.md (hunk 4 above)
Comment (msiron-entalpic) [reply lematerial.md:229]: sounds good to me!

**Review by lvwerra — APPROVED** [review]
Looks good to me - I'll let you resolve the comments with @clefourrier and merge when you feel ready!

File: lematerial.md (hunk 3 above)
Comment (clefourrier) [reply lematerial.md:219]: Ok, sgtm

File: lematerial.md (hunk 2 above)
Comment (inelgnu) [inline lematerial.md:74]:
```suggestion
| v.1.0 | <ul><li><strong>Data collection and merging</strong> from various sources: Materials Project, Alexandria and OQMD datasets, including multiple fields and DFT functionals (PBE, PBESol, SCAN). </li><li><strong>Data cleaning</strong>: identify and remove datapoints not conforming to the set standard (e.g. with non-compatible calculations). </li><li><strong>Standardization</strong>: uniformly format the set of fields across databases, using Optimade standard </li><li><strong>Material fingerprint</strong>: attribute a unique identifier to each material with a well benchmarked hashing function, enabling to remove duplicates</li><li><strong>Accessibility and visualization</strong>: the proposed new datasets can be visualized thanks to open source tools (phase diagrams, material explorer) based on Crystal Toolkit, Pymatgen, and Dash components developed by Materials Project </li></ul>  | Dec. 10, 2024 |
```

**Pull request merged**